edk2toolext/environment/extdeptypes/git_dependency.py

```diff
@@ -65,8 +64,11 @@ def __str__(self):
 
     def fetch(self):
         """Fetches the dependency using internal state from the init."""
-        # def resolve(file_system_path, dependency, force=False, ignore=False, update_ok=False):
-        repo_resolver.resolve(self._local_repo_root_path, self._repo_resolver_dep_obj, update_ok=True)
+        try:
+            repo_resolver.resolve(self._local_repo_root_path, self._repo_resolver_dep_obj, update_ok=True)
+        except repo_resolver.GitCommandError as e:
+            logging.debug(f'Cmd failed for git dependency: {self._local_repo_root_path}')
+            logging.debug(e)
 
```

Comment:
This does not change the flow. The previous version of `repo_resolver.resolve()`  relied on `RunCmd()` which returns the return value of the command executed; The new version uses GitPython which raises an exception in the same scenario (if the command returns a non-zero). Since we don't check the output for resolve in the old version, I opted to continue on error, only that we now catch the exception and do a bit of extra logging.